capa/ida/plugin/form.py

```diff
@@ -525,8 +525,7 @@ def ida_hook_rebase(self, meta, post=False):
             meta["prev_base"] = idaapi.get_imagebase()
             self.model_data.reset()
 
-    def load_capa_rules(self):
-        """load capa rules from directory specified by user, either using IDA UI or settings"""
+    def ensure_capa_settings_rule_path(self):
```

Comment:
split this logic out to keep functions shorter and easier to read. no change in functionality.